src/main/scala/modbat/mbt/ModelInstance.scala

```diff
@@ -140,6 +148,16 @@ class ModelInstance (val mbt: MBT, val model: Model,
     mbt.launchedModelInst += model
     currentState = initialState
     StateCoverage.cover(initialState)
+
+    // get first instance of model to add the graph to it
+//    val firstModelInstance: ModelInstance = mbt.firstInstance.getOrElse(this.className, sys.error("Illegal state"))
+    if (isFirstInstance) {
+      val graph: GraphAdaptor = new GraphAdaptor(mbt.config, this)
+      graph.printGraphTo(mbt.config.dotDir + File.separator + this.className + "_graph.dot")
```

Comment:
This will throw an uncaught exception inside `getPrintStream` in `Graph.java`.
Try to change the code such that (1) a warning or error message is shown if the file cannot be opened (e.g., because `mbt.config.dotDir` is set incorrectly) and (2) perhaps is able to continue without writing the graph but also without crashing.

Reply:
Regarding (1): I followed the same convention as in `PathVisualizer` [here](https://github.com/cyrille-artho/modbat/blob/00fb6304724a983339dda05b02df689f00466c40/src/main/scala/modbat/mbt/PathVisualizer.scala#L16)

Regarding (2): I fixed the code in `addAndLaunch` method [here](https://github.com/cyrille-artho/modbat/blob/67a182400c874b6a7bd2f06294ac0ec716277453/src/main/scala/modbat/mbt/ModelInstance.scala#L164) that the code does not exit even if `dotDir` does not exist. However, the problem still lies in `PathVisualizer` [here](https://github.com/cyrille-artho/modbat/blob/00fb6304724a983339dda05b02df689f00466c40/src/main/scala/modbat/mbt/PathVisualizer.scala#L16), which needs a bit of re-factoring. This PR does not change `PathVisualizer`. Maybe an issue should be created to do this refactoring in that case. I wrote a regression test in `bin/test.sh` that checks that the JVM exits with status `1` if `--dotify-path-coverage` is activated  but the directory specified by `--dot-dir` does not exist.